main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"code.cloudfoundry.org/cli/plugin"
+	"github.com/spf13/cobra"
+)
+
+var (
+	Verbose  bool
+	shutdown chan struct{}
+	Stdin    = os.Stdin
+	Stdout   = os.Stdout
+	Stderr   = os.Stderr
+)
+
+func init() {
+	shutdown = make(chan struct{})
+	go func() {
+		sig := make(chan os.Signal, 3)
+		signal.Notify(sig, syscall.SIGINT, syscall.SIGQUIT, syscall.SIGTERM)
+		<-sig
+		close(shutdown)
+		for range sig {
+			log.Println("...shutting down")
+		}
+	}()
+	os.Stdin = nil
+	os.Stdout = nil
```

Comment:
This looks like a bit of a smell to me. What's the purpose of overriding these?

Reply:
It's not ideal.

Since the plugin is most useful when it can be used as a standard unix-style cli tool piping/redirecting output etc I wanted to be _sure_ that only known output was written (ie `cf conduit pg -- pg_dump > backup.sql` should be guarenteed to only contain dump output) 

The plugin framework/code likes to print things to Stdout, and does not give a way to alter this behaviour, so I took away it's references to Stdout.

This was more of an issue when using the plugin helpers to communicate so it's possible the original issue this was working around has been mitigated by the changes to how `cf` commands are called.

I will revisit and tidy up if possible.

Reply:
This is addressed. Now uses the API directly so no need for the hacky Stdout hijacking.